shared/js/background/atb.es6.js

```diff
@@ -173,18 +176,43 @@ const ATB = (() => {
             // - the user wasn't already looking at the app install page
             // - the user hasn't seen the page before
             settings.ready().then(() => {
-                chrome.tabs.query({currentWindow: true, active: true}, function (tabs) {
-                    const domain = (tabs && tabs[0]) ? tabs[0].url : ''
-                    if (ATB.canShowPostInstall(domain)) {
-                        settings.updateSetting('hasSeenPostInstall', true)
-                        let postInstallURL = 'https://duckduckgo.com/app?post=1'
-                        const atb = settings.getSetting('atb')
-                        postInstallURL += atb ? `&atb=${atb}` : ''
-                        chrome.tabs.create({
-                            url: postInstallURL
+                // Special case for the cross product promotion on desktop (cppd) experiment
+                if (settings.getSetting('isMultiStepOnboarding')) {
+                    // We find the tab containing the cross product promotion modal
+                    // and inject a content script that will notify the page that the
+                    // extension was successfully installed
+                    // Note: that we don't know on which window that tab is
+                    chrome.tabs.query({}, (tabs) => {
+                        const tab = tabs.find((tab) => {
+                            const { hostname, searchParams } = new URL(tab.url)
+
+                            return (hostname.includes('duckduckgo.com') && !!searchParams.get('natb'))
```

Comment:
We might want to make this more explicit in production and specify the url must be `https://duckduckgo.com`

Otherwise something like `https://not-duckduckgo.com?natb=1` could trigger this

Reply:
Good point, we have https://www.npmjs.com/package/psl in the package.json so I can use that and check on the domain rigorously

Reply:
actually `psl` is only used for the script so will just do simple check.

Reply:
should be fixed by https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/452/commits/1ed4e4afb423f609e0bb40ec17c2e9718f4ed646